flagutil/flagutil.go

```diff
@@ -0,0 +1,75 @@
+// Package containing flag utilities. This includes implementations of flag.Value and flag.Getter interfaces. Also
+// contains methods like ValidateFlags.
+package flagutil
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+)
+
+type flagError string
```

Comment:
It might be clearer to make `flagError` a struct:
```go
type FlagError struct {
        flag *flag.Flag, // not sure what the best name for this field would be
        err error  // same here
}
```
(I'd export it too, seems like it might be something other packages would want to use)

then you can have:

```go
func (f *flagError) Error() string {
        return fmt.Sprintf("Flag: %s, Error: %s, Usage: %s", f.flag.Name, f.err.Error(), f.flag.Usage)
}
```

What do you think?